apicurio/registry.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: apicurio-registry
+  description: The Apicurio Registry API V2
+spec:
+  type: openapi
+  lifecycle: production
+  owner: apicurio
+  definition:
+    $text: https://apicurio-registry-mt-apicurio-apicurio-registry.apps.smaug.na.operate-first.cloud/api-specifications/registry/v2/openapi.json
```

Comment:
Can we use github link from #2

Reply:
Do you mind elaborating on the reason?
This link refers to the registry instance deployed in the environment, I used it as it's the "most correct" and "closest" source of truth at best of my understanding.

Reply:
I think my requirements have been oudated. This backstage is no longer app-services effort. It is more to fit some apicurio team needs. Sorry for that